packages/enzyme-adapter-react-helper/src/getAdapterForReactVersion.js

```diff
@@ -0,0 +1,32 @@
+import semver from 'semver';
+
+export default function getAdapterForReactVersion(reactVersion) {
+  const normalizedVersion = semver.coerce(reactVersion);
```

Comment:
Need this because something like "16.3" isn't a valid version, per semver.

Their docs state that `coerce` is guaranteed to generate a version as long as there is a number in the string.